health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,325 @@
+          size={"large"}
+          variation={"primary"}
+          label={t("Close")}
+          onClick={() => {
+            setShowPopup(false);
+          }}
+          style={{ width: "12.5rem" }}
+        />,
+      ]}
+      sortFooterChildren={true}
+    >
+      <ViewMap lat={bounds?.latitude} long={bounds?.longitude} type={type}></ViewMap>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)